client/src/app/+videos/+video-watch/video-watch.component.ts

```diff
@@ -219,6 +219,12 @@ export class VideoWatchComponent implements OnInit, OnDestroy {
     return $localize`You need to be <a href="/login">logged in</a> to rate this video.`
   }
 
+  getExactNumberOfViews () {
+    return (this.video.views >= 1000)
+      ? (this.video.views + ' ' + (this.video.isLive ? 'views' : 'viewers'))
```

Comment:
I think the ternary operation here is wrong. If the video is a live, then 'viewers' should be displayed rather than 'views', and vice versa.

Reply:
You're right, my bad

Reply:
I make the change